spec/index.md

```diff
@@ -2340,6 +2340,30 @@ This function returns fee percentiles, measured in millisatoshi/vbyte (1000 mill
 
 The [standard nearest-rank estimation method](https://en.wikipedia.org/wiki/Percentile#The_nearest-rank_method), inclusive, with the addition of a 0th percentile is used. Concretely, for any i from 1 to 100, the ith percentile is the fee with rank `⌈i * 100⌉`. The 0th percentile is defined as the smallest fee (excluding coinbase transactions).
 
+### IC method `fetch_logs` {#ic-fetch_logs}
+
+Provides the logs of the canister including trap messages.
+Logs are stored in a circular buffer of 4KiB.
```

Comment:
Do the `4KiB` also include storage overhead of the underlying data structure? Ideally that's not the case and we can say that the total length of all log contents is at most `4KB`. Then we should drop also the reference to the "circular buffer" and instead say that oldest logs get dropped if appending new logs would exceed the log memory capacity.

Reply:
Done.